### DISCUSSION TITLE
[Keycloak] Server error There is a problem with the server configuration.  Check the server logs for more information.

### DISCUSSION THREAD

**Discussion by nguyenhuyhoang241298** [nextauthjs/next-auth#10453] [Help]
Hello. I have 2 applications with 2 other next auth's version. 
With next-auth 4.24.5, everthing is ok. (the first application.)

![image](https://github.com/nextauthjs/next-auth/assets/113531382/c73b001a-59f2-49b3-bd5f-47f6f04ad10b)

But in the second application with next-auth 5.0.0-beta.15, it's not working even when I copy all the config in the first application.

![image](https://github.com/nextauthjs/next-auth/assets/113531382/eb5b6175-0b76-42cc-99c2-afc195ca6860)

Here is my config:
![image](https://github.com/nextauthjs/next-auth/assets/113531382/2053b863-4c8c-4bf1-a3e3-aa8a19fea641)

![image](https://github.com/nextauthjs/next-auth/assets/113531382/1a8a2d57-9e94-42f5-8c79-ce866649aeee)

And this is errors:

![image](https://github.com/nextauthjs/next-auth/assets/113531382/c636082e-3952-4707-8986-cefe44a6d3cd)

![image](https://github.com/nextauthjs/next-auth/assets/113531382/41a1ce11-1177-438b-b159-888ec0912e97)

![image](https://github.com/nextauthjs/next-auth/assets/113531382/9151bfa5-5ec2-4b9e-bb5b-3f0c9edddef4)

Many thanks!

**Comment by nguyenhuyhoang241298** [comment]
This is my keycloak config
![image](https://github.com/nextauthjs/next-auth/assets/113531382/48f054bb-275f-4942-b148-1f5d312bb114)

**Comment by ClydeFrog04** [comment]
@nguyenhuyhoang241298 Did you ever figure this one out?
